react/admin/utils/messages.ts

```diff
@@ -336,6 +336,30 @@ export const costCenterMessages = defineMessages({
   marketingTags: {
     id: `${adminPrefix}costCenter-details.marketing-tags`,
   },
+  nothingToShow: {
+    id: `${adminPrefix}costCenter-details.addresses.nothing-to-show`,
+    defaultMessage: 'Nothing to show',
```

Comment:
Not sure if you need the `defaultMessage` here, context should handle this just fine